### DISCUSSION TITLE
The structure should become stable?

### DISCUSSION THREAD

**Discussion by Prutthi** [SSCHAcode/python-sscha#175] [Q&A] [answered]
Dear expert SSHCA code,

Regarding the SSCHA study, I am intrigued to understand the scenario where there is an unstable structure under high pressure. Upon investigating this structure using the SSCHA code through structure relaxation at various temperatures, the expectation is that the structure should become stable. If it doesn't, the issue might be related to the settings in the input file of the SSCHA code. Assuming this is the case, what corrections should be made? Is it dependent on parameters such as max_pop and N_configs?

Best,
Prutthipong

**Comment by mesonepigreco** [comment, marked as answer]
If the structure is physically unstable, it will be unstable also for the SSCHA. The structure with the SSCHA becomes stable only if anharmonicity and quantum/thermal fluctuations are strong enough to stabilize it (they could also destabilize an unstable structure, e.g., at high temperatures a solid melts).

If a structure should be stabilized by anharmonicity but is not in an SSCHA calculation, one should study the convergence with respect to the number of configurations and, more importantly, the dimension of the q-mesh (supercell) employed in the calculation. The latter may have a big impact on the role played by quantum/thermal anharmonic fluctuations, and should be appropriately converged.

Bests,
Lorenzo
